src/test/java/com/github/checkstyle/patchfilter/SuppressionPatchFilterTest.java

```diff
@@ -50,34 +57,181 @@ public void testAccept() throws Exception {
 
     @Test
     public void testMultiChangesOnOneFileOne() throws Exception {
-        final String fileName = getPath("MultiChangesOnOneFilePatch.txt");
```

Comment:
please keep SuppressionPatchFilterTest.java  as is and move all test for patch parsing to separate test class UnifiedDiffReaderTest

Reply:
done